examples/dynamo/vgg16_fp8_ptq.py

```diff
@@ -111,7 +111,12 @@ def vgg16(num_classes=1000, init_weights=False):
     type=int,
     help="Batch size for tuning the model with PTQ and FP8",
```

Comment:
The filename, docstring, and argument description probably need to be changed, since it's not FP8-only quantization now.